CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
-## 0.7.2 (2017-07-04)
+<a name="0.7.3"></a>
```

Comment:
FYI something went wrong when you tried to rebase.  Its counting all of the existing commits as new.

Reply:
that's why I deleted and recreated my fork. I'll create a new PR when I'm ready, this one will be cancelled I think

Reply:
Ok.  Sorry you've run into these problems.  Too bad our time zones are so far off (technically at work right now) otherwise we could get on gitter at the same time to try to work through them to make things easier (I presume).